src/routes/+layout.svelte

```diff
@@ -0,0 +1,118 @@
+<script lang="ts">
+  import { Content, Grid, Theme, Row, Column } from 'carbon-components-svelte'
+
+  import '../global.css'
+  import Header from '$components/Header.svelte'
+  import Footer from '$components/Footer.svelte'
+  let theme: 'white' = 'white' as const
+</script>
+
+<Theme bind:theme />
+
+<Header />
+<Content>
+  <Grid>
+    <Row>
+      <Column>
+        <slot />
+      </Column>
+    </Row>
+  </Grid>
+</Content>
+<Footer />
+
+<style>
+  /* Styles to normalize how the markdown-sourced content looks */
```

Comment:
should these be added to the `global.css` file instead?

Reply:
Yep, that would be a better place. Unfortunately, `global.css` is currently a blob of compiled CSS pulled directly from the [Carbon Components source code](https://github.com/carbon-design-system/carbon-components-svelte/blob/master/css/white.css), which will make it a bit awkward to move things there.

We're planning to refresh the design in this project. Let's look at moving the styles over at then.